Assets/Mirror/Components/NetworkAnimator.cs

```diff
@@ -194,6 +204,11 @@ void HandleAnimMsg(int stateHash, float normalizedTime, int layerId, NetworkRead
                 animator.Play(stateHash, layerId, normalizedTime);
             }
 
+            if (animator.GetLayerWeight(layerId) != weight)
```

Comment:
Do we need to check if it is changed? or can we just call set?

Reply:
I'm not exactly sure what the depths of the Unity Animator does if you fiddle with the layer weight every frame, but I don't imagine it's a problem to save on the get + compare, so I'll push a new commit with this removed.

Reply:
I've just double checked and both the Get and Set are `extern` calls so its probably best to just call the set.

It might be worth doing the weight in its own Cmd/Rpc so that it is only set when it changes, but I can try that out and do that in a later PR.

Reply:
Agreed, I've removed the get in 9d35630